custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 102
+      name: brightness
+      type: integer
+      range:
+        min: 1
+        max: 10
+
+secondary_entities:
+  - entity: button
+    name: Reset Filter Alarm
```

Comment:
Change "Alarm" to life" here to connect it logically to the sensor below.  Including "Alarm" makes it seem like it is connected to an alarm bell.

Reply:
Agree i can change the name